src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Game.java

```diff
@@ -64,4 +65,14 @@ public Game getNextGame() {
   public Collection<Match> getMatches() {
     return matches;
   }
+
+  @JsonIgnore
+  public Match getPlayingMatch() {
+    for (Match match : matches) {
```

Comment:
this assumes that only one match is in state playing.
Maybe this overlaps with this PR: #113 and you can use stuff from there...

Reply:
Schaue ich mir noch an.

Reply:
Issue opened: https://github.com/sopra-fs22-group-36/screw-your-neighbor-server/issues/119

Reply:
Es wird pro Game auch nur einen Match im Status PLAYING geben?